packages/FAQ/docs/creating-a-smart-invoice.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Creating a Smart Invoice
+--- 
+
+## Step 1:  Project Details 
+
+Here you will enter your project’s details in sequential order: 
+
+1. Project Name or ID
```

Comment:
If we're going to list out the fields in the app, I think we should at least explain what they are. As is its not information a user can't get from simply using the app.